app/models/developers/external_profiles/linkedin_profile_fetcher.rb

```diff
@@ -0,0 +1,47 @@
+module Developers::ExternalProfiles
+  class LinkedinProfileFetcher
+    def developer_profiles
+      profiles = []
+      developers_with_linked_in_profiles.each do |developer|
```

Comment:
```suggestion
      developers_with_linkedin_profiles.each do |developer|
```

To keep our LinkedIn naming consistent.